include/quic/udp.hpp

```diff
@@ -38,17 +38,27 @@ namespace oxen::quic
                 ;
 
         using receive_callback_t = std::function<void(const Packet& pkt)>;
+        using receive_batch_callback_t = std::function<void()>;
```

Comment:
I have a knee-jerk reaction to suffixing with `{}_t` now and you only have yourself to blame